core/unix/inc/TUnixSystem.h

```diff
@@ -71,6 +90,12 @@ class TUnixSystem : public TSystem {
    static int          UnixRecv(int sock, void *buf, int len, int flag);
    static int          UnixSend(int sock, const void *buf, int len, int flag);
 
+   // added helper static members for stacktrace
+   static char *const                  kStackArgv[];
```

Comment:
reduce the spacing between const and kStack to the minimal number to align the group of variable/functions names.